freqtrade/plugins/pairlist/StaticPairList.py

```diff
@@ -50,9 +50,10 @@ def gen_pairlist(self, cached_pairlist: List[str], tickers: Dict) -> List[str]:
         :return: List of pairs
         """
         if self._allow_inactive:
-            return self._config['exchange']['pair_whitelist']
+            return self.verify_whitelist(self._config['exchange']['pair_whitelist'], logger.info)
```

Comment:
I think this is a special case (when `allow_inactive = True`), in which we'd not wan't to expand the whitelist.

Problem here is, the whitelist expansion implicitly checks if the pair exists.
This flag was however explicitly added in #3939 to allow backtesting no longer existing pairs, which this will basically prevent.

Obviously, we'd need to add a note (`pairlist.md`) that this will prevent whitelist expansion to the docs ...

Reply:
From there, I called expand_pairlist with parameter keep invalid = True, so that invalid pairs are maintained in the whitelist.

Reply:
Great way to solve this! :+1: